config/routes.rb

```diff
@@ -289,5 +289,6 @@
   devise_scope :user do
     get 'avatar/:id/:style' => 'users/registrations#avatar', as: 'avatar'
     post 'avatar_signature' => 'users/registrations#signature'
+    get 'auth_token_sign_in' => 'users/sessions#auth_token_create'
```

Comment:
I know I'm nitpicking, but the normal `sign_in` is located at `users/sign_in`. I believe this route should also be at `users/auth_token_sign_in`.